source/support.tex

```diff
@@ -2127,7 +2127,7 @@
 terminates by throwing an exception~(\ref{expr.new}).
 \end{itemdescr}
 
-\indexlibrary{\idxcode{new}!\idxcode{operator}}%
+\indexlibrary{\idxcode{delete}!\idxcode{operator}}%
```

Comment:
I believe this is consistent with the other index entries for array-delete, and did not want to be too creative at this point.  I can consistently index 'delete[]' if preferred?